### PR TITLE
Examples: Improve molecule demos.

### PR DESCRIPTION
**Pull request by Mugen87**
Related issue: -

**Description**

Improves the usability of molecule examples by using a `lil-gui` interface.

### CODE REVIEW

**Comment by Mugen87** [comment]
https://raw.githack.com/Mugen87/three.js/dev67/examples/webgl_loader_pdb.html

File: examples/webgl_loader_pdb.html [hunk 1]
```diff
@@ -137,46 +115,26 @@
...
-					menu.appendChild( button );
-
-					const url = 'models/pdb/' + MOLECULES[ m ];
+				//
 
-					button.addEventListener( 'click', generateButtonCallback( url ) );
+				const gui = new GUI();
 
-				}
+				gui.add( params, 'molecule', MOLECULES ).onChange( loadMolecule );
```
Comment (mrdoob) [inline examples/webgl_loader_pdb.html:128]: I think instead of `'molecule'` it should be `'compound'`?

Comment (Mugen87) [reply examples/webgl_loader_pdb.html:128]: Should I update then all places where `molecule` is used as a term? 

BTW: I think something like Ethanol is actually a molecule^^. `compound` seems to be just a synonym.

Comment (mrdoob) [reply examples/webgl_loader_pdb.html:128]: Oh true, they're molecules. I got confused sorry 😅

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**